ts/features/fims/history/store/selectors/index.ts

```diff
@@ -29,8 +29,8 @@ export const fimsHistoryErrorSelector = (
 export const fimsIsHistoryEnabledSelector = (state: GlobalState) =>
   pipe(
     state,
-    backendStatusSelector,
-    O.map(backendStatus => backendStatus.config.fims.historyEnabled !== false),
+    remoteConfigSelector,
+    O.map(c => c.fims.historyEnabled !== false),
```

Comment:
Should we use `remoteConfig` instead of `c`, to improve readability?

Reply:
```suggestion
    O.map(remoteConfig => remoteConfig.fims.historyEnabled !== false),
```